packages/opentelemetry-core/src/common/ConsoleLogger.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Logger } from '@opentelemetry/types';
+import * as consoleLogLevel from 'console-log-level';
+import * as util from 'util';
+
+/** Defines a logger options interface. */
+export interface LoggerOptions {
+  /** level of logger - 0:disable, 1: error, 2: warn, 3: info, 4: debug  */
+  level?: string;
+}
+export type ConsoleLogLevel = 'error' | 'warn' | 'info' | 'debug';
+export type LogLevel = 'silent' | ConsoleLogLevel;
+
+/** The list of log levels. */
+export const LEVELS: ReadonlyArray<LogLevel> = [
+  'silent',
```

Comment:
The most popular loggers in Node doesn't have a silent `level`.
It can be problematic from the ergonomic point of view:
- https://github.com/pinojs/pino/blob/master/docs/api.md#logger
- https://www.npmjs.com/package/winston#logging-levels
- https://github.com/trentm/node-bunyan#levels

I `error`, `warn`, `info` and `debug` are common.
I'd vote for removing silent.

Reply:
Hmm, https://www.npmjs.com/package/console-log-level also doesn't have a `silent` level, do I miss something?

Reply:
Conceptually the way you get `silent` is to use the NoOpLogger I'd think anyway